Traceback (most recent call last):
  File "RandomGreedy.py", line 71, in <module>
    random_worker(default_dqn_config.config)
  File "RandomGreedy.py", line 56, in random_worker
    assert legal_actions[action]
AssertionError
